images/runtime/dotnetcore/7.0/base.buster.staging.Dockerfile

```diff
@@ -0,0 +1,67 @@
+# syntax=docker/dockerfile:1.3
```

Comment:
Visual representation of the changes between this file and the non-staging dockerfile
![image](https://user-images.githubusercontent.com/107068277/199034547-86edf9cf-45b7-43d0-998c-1069cd936467.png)

Reply:
Appreciate it! Thanks